src/extension/src/scripts/run-tests.ts

```diff
@@ -0,0 +1,150 @@
+let child_process = require("child_process");
+const path = require("path");
+const fs = require("fs");
+
+const testFolder = path.join(__dirname, "..", "..", "src", "template_test");
+console.log(testFolder);
+let files: string[] = [];
+
+//Recursively delete all generated projects in template_test
+let deleteFolderRecursive = function(pathname: string) {
+  let files = [];
+  if (fs.existsSync(pathname)) {
+    files = fs.readdirSync(pathname);
+    files.forEach(function(file: string, index: number) {
+      var curPath = path.join(pathname, file);
+      if (fs.lstatSync(curPath).isDirectory()) {
+        deleteFolderRecursive(curPath);
+      } else {
+        // delete file
+        fs.unlinkSync(curPath);
+      }
+    });
+    fs.rmdirSync(pathname);
+    console.log(`deleted ${pathname}`);
+  }
+};
+
+fs.readdirSync(testFolder).forEach((file: string) => {
+  files.push(file.toString());
+});
+
+function kill(pid: any) {
+  try {
+    child_process.execSync("taskkill /PID " + pid + " /T /F", function(
+      error: any,
+      stdout: any,
+      stderr: any
+    ) {
+      console.log("Stdout from kill: " + stdout);
+      console.log("Stderr from kill:" + stderr);
+      if (error !== null) {
+        console.log("Error from kill: " + error);
+      }
+    });
+  } catch (err) {
+    console.log(`Error from taskkill: ${err}`);
+  }
+}
+
+function sleep(ms: number) {
```

Comment:
Side note: Feel like we should add this to a utils class or something, see it being used in many places now.

Reply:
Hmm ok, I can create it in another pr